tonic/dart_state.h

```diff
@@ -21,7 +24,17 @@ class DartState : public tonic::DartState {
   static DartState* From(Dart_Isolate isolate);
   static DartState* Current();
 
+  const ftl::RefPtr<ftl::TaskRunner>& ui_task_runner() {
```

Comment:
Having the UI and IO task runners hosted in the "DartState" seems a but awkward.